tests/functional/shutdown/10-no-port-file/flow.cylc

```diff
@@ -0,0 +1,13 @@
+[cylc]
+    [[events]]
+        abort on stalled = False
+        abort on timeout = True
+        timeout = PT1M
+
+[scheduling]
+    [[graph]]
+        R1 = t1
+
+[runtime]
+    [[t1]]
+        script = false
```

Comment:
I don't think this should be here.

Reply:
Good spot, the deletion must have been accidentally overridden when merging master into the branch.

Took a bit of googling to figure out how to get `git log` to show me the files that were deleted on master:
```
git log --diff-filter=D --stat af1765ceeae5b921e181b59d988002d7c2528a74..master
```